pkg/api/router.go

```diff
@@ -51,15 +45,26 @@ func (s *Service) MountDebug() {
 		web.NoCacheHeadersHandler,
 		web.FinalHandler(s.router),
 	)
+
+	for _, o := range opts {
+		o(s)
+	}
 }
 
-func (s *Service) MountAPI() {
-	if s.router == nil {
-		s.router = mux.NewRouter()
-		s.router.NotFoundHandler = http.HandlerFunc(jsonhttp.NotFoundHandler)
+// WithFullAPI will enable all available endpoints, because some endpoints are not available during syncing.
+func WithFullAPI() MountOption {
```

Comment:
why have an option if we can explicitly enable the the full API using `EnableFullAPI()`

Reply:
Initial idea to provide options is for unit testing purposes. Was considering to add `WithFullAPIDisabled` option as well. And then expand `testServerOptions` in api_test.go. 
Also it can be used in one line to mount and enable `apiService.Mount(api.WithFullAPI())`, which is the case in devnode.go